rpc/src/v1/helpers/subscribers.rs

```diff
@@ -78,7 +79,8 @@ impl<T> Default for Subscribers<T> {
 
 impl<T> Subscribers<T> {
 	fn next_id(&mut self) -> Id {
-		let data = H64::random_using(&mut self.rand);
+		let mut data = H64::default();
+		self.rand.fill_bytes(&mut data.as_bytes_mut());
```

Comment:
ahh all right, so we should be able to revert this once both are rand 0.7 then?

Reply:
I'm out of the loop, forgive the silly q: How does this change with `rand 0.7`?

Reply:
This is related to the changes of `OsRng` where version `0.6` is not compatible with version `0.7`. I don't really know exactly what was caused it but I think the `trait OsRngImpl` was removed in `0.7` when `OsRng` started to depend on `getrandom` crate rather than implement it.

Thus, because we now use `rand 0.7` in `ethereum-types v0.8` and `rand 0.6` in `rpc` it `let data = H64::random_using(&mut self.rand);` doesn't work anymore.

Reply:
ok, let's add a `todo` here so we don't forget.

Reply:
Yes, @niklasad1 is right. I'll add todo here